### ISSUE TITLE
front: map: LGV Signals are not properly displayed

### ISSUE THREAD

**Issue by nicolaswurtz** [OpenRailAssociation/osrd#4901]
As the title says...

![Image](https://github.com/osrd-project/osrd/assets/6003856/ef742379-aac3-4865-b08d-04e2203e5bef)

**Comment by sim51** [comment]
@nicolaswurtz  I don't really know how to display those signals, but by adding a `case 'REP TGV':` here https://github.com/osrd-project/osrd/blob/dev/front/src/common/Map/Layers/geoSignalsLayers.ts#L448 , I have something better.

Code extract : 
```
switch (type) {
    case 'REPER VIT':
    case 'REP TGV':
      return getSignalEmptyLayerProps(context, type, iconOffset, 'extensions_sncf_label');
   ....
```

![image](https://github.com/osrd-project/osrd/assets/231657/d2dc3407-c127-456e-b8e6-3645f21a0370)

I don't know if it's the desired design, or if we have to write a new  `getSignalXXXLayerProps` function.

**Comment by clarani** [comment]
Solved by #5166
